src/Native/Bootstrap/main.cpp

```diff
@@ -372,6 +372,11 @@ extern "C" void RhpBulkWriteBarrier()
 {
     throw 42;
 }
+
+extern "C" Object* RhNewArrayAsString(MethodTable*)
```

Comment:
This is bug that I have introduced recently. Could you please change the following line in CppWriter from:

ecmaMethod.GetAttributeStringValue("System.Runtime", "RuntimeImport")

to:

ecmaMethod.GetAttributeStringValue("System.Runtime", "RuntimeImportAttribute")